client/components/data/query-whois/index.jsx

```diff
@@ -29,7 +29,10 @@ QueryWhois.propTypes = {
 	requestWhois: PropTypes.func.isRequired
 };
 
+const mapStateToProps = ( state, { domain } ) =>
+	( { requesting: isRequestingWhois( state, domain ) } );
```

Comment:
@deBhal: see my [reply on the original thread](https://github.com/Automattic/wp-calypso/pull/14701#discussion_r122165440). You can totally keep this as it were — inlining the selector inside `connect` — or break it apart, but it's totally your choice. Again, apologies. 🙇